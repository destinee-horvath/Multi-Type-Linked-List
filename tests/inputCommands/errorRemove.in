REMOVE 
REMOVE -1
REMOVEe
remove 
REMOVE abc