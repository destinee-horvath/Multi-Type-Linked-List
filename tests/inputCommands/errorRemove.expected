INVALID COMMAND: REMOVE
INVALID COMMAND: REMOVE
INVALID COMMAND: REMOVE
INVALID COMMAND: INPUT
INVALID COMMAND: REMOVE
